### ISSUE TITLE
Handle `super` in arrow functions

### ISSUE THREAD

**Issue by overlookmotel** [overlookmotel/livepack#306]
Input:

```js
class D {
  foo() { return 1; }
}

class C extends D {
  bar() {
    return () => super.foo();
  }
}

const c = new C();
export default c.bar();
```

This fails to serialize with error from Babel parser `SyntaxError: 'super' is only allowed in object methods and classes`.

### Cause + solution

This is caused by trying to parse `() => super.foo()` alone when serializing. This is not valid JS.

Needs to be wrapped in an object method in to be valid JS e.g. `({x() { () => super.foo() }})`.

Would need to record that function contains `super` in tracker comment so know to add this wrapper before parsing.

### Additional problem

Output created by Babel plugin does not record usage of `super` in the arrow function.

```js
bar() {
  livepack_tracker(
    /*livepack_track:{"id":6,"scopes":[{"blockId":5,"varNames":["C"],"constNames":["C"],"blockName":"C"}],"isStrict":true,"isProtoMethod":true,"superVarName":"C"}*/
    () => [[livepack_scopeId_1, C]]
  );
  return () => {
    livepack_tracker(
      /*livepack_track:{"id":7,"scopes":[],"isStrict":true}*/
      () => []
    );
    return super.foo();
  };
}
```

**Comment by overlookmotel** [comment]
There is a bug in current implementation. `this` also needs to be in scope for the transpiled `super` to operate correctly.

**Comment by overlookmotel** [comment]
Fixed in 3bc4d81.
